pkg/config/env_var.go

```diff
@@ -27,6 +28,16 @@ func (evl EnvVarList) ToStringSlice() []string {
 
 	return envSlice
 }
+func (evl EnvVarList) ToDevfileEnv() []common.Env {
```

Comment:
Please add comments for this and the functions below

Reply:
added